src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -30,8 +30,20 @@
       <RunTimeDependencyCopyLocal Include="@(RunTimeDependencyCopyLocalFile -> '%(File)')"  />
       <RunTimeDependencyCopyLocal Include="$(TargetingPackPath)/*" />
       
-      <RunTimeArtifactsExcludeFiles Include="protononjit.dll" />
-      <RunTimeArtifactsExcludeFiles Include="linuxonjit.dll" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/createdump.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/dbgshim.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/ilasm.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/ildasm.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/linuxonjit.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/mcs.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/mscordaccore.pdb" />
```

Comment:
Can we drop the DAC/DBI symbols and still get SOS working on managed dumps from CI?

cc: @hoyosjs

Reply:
I think we should keep the DAC/DBI symbols just in case. The rest of the ones listed here we can drop IMO.

Reply:
The DAC/DBI pdb's are not needed. The debuggee will never have any of their frames in the callstack (only a debugger is expected to have such frames).

Reply:
I don't think you should exclude ilasm/ildasm -- they are product binaries.

So what is affected by this? E.g., what can I not do with the exclusion to the non-default JIT and SuperPMI ones?

Reply:
E.g., superpmi.pdb and superpmi-shim-collector.pdb are PDBs associated with EXEs that run during the pri-0 test run. So will we not get core dumps if they fail?

Reply:
I have double-checked with @noahfalk that PDB's are not needed for producing crash dumps. For symbolic Watson traces they probably would be necessary (adding another 30 MB to the correlation payload).

Reply:
Do we have any testing that would use ilasm/ildasm?

Reply:
I think though that I agree that the pruning should be distinguished between the ci and the dev loop. Seems like we can be more careless with size in the core_root for dev usage.

Although it would be nice to come to a minimal set of requirements for our core_root for both ci and dev.

Reply:
> Do we have any testing that would use ilasm/ildasm?

Unsure, but if not, we should

We used to have ildasm/ilasm round-trip testing, but I think we've lost it.

Reply:
Right, this was lost in the move to azdo.

Reply:
Can I suggest that we maybe have a flag `MinimalCoreRoot`. Which we can set in the CI. We have no reason to include ildasm/ilasm in our helix submissions, and the day we re-create the il* roundtrip tests those can include the ilasm/ildasm binaries and pdbs.